e2e-tests/src/config.rs

```diff
@@ -1,9 +1,23 @@
+use std::env;
+
 use aleph_client::{RootConnection, SignedConnection};
 use clap::{Args, Parser};
+use once_cell::sync::Lazy;
 use primitives::SessionIndex;
 
 use crate::accounts::{get_sudo_key, get_validators_keys, get_validators_seeds, NodeKeys};
 
+static GLOBAL_CONFIG: Lazy<Config> = Lazy::new(|| {
```

Comment:
You mean for the name of the variable? I disagree, so ignoring, since it's a nit :) (Reasoning: `E2E` is implicit, since we're in the `e2e` package, `global` introduces new information)